imports/plugins/included/staticPagesStart/server/policy.js

```diff
@@ -0,0 +1,9 @@
+import { BrowserPolicy } from "meteor/browser-policy-common";
+/*
+ * set browser policies
+ */
+BrowserPolicy.content.allowOriginForAll("*.facebook.com");
```

Comment:
Any reason why we're using these policies? I think we already have these dependencies in our project.